### ISSUE TITLE
Training with Cityscapes datasets

### ISSUE THREAD

**Issue by sazippe** [xy-guo/Learning-Monocular-Depth-by-Stereo#10]
Table 1
StereoUnsupFt→Mono pt No S,K→K 0.099 0.745 4.424 0.182 0.884 0.963 0.983
StereoUnsupFt→Mono pt No S,K→C,K 0.095 0.703 4.316 0.177 0.892 0.966 0.984

From the result in Table1, i can see that using Cityscapes datasets will improve overall performance.

I want to know how to use Cityscapes datasets on training.
After training mono network by KITTI datasets, then finetune its network by Cityscapes datasets?
Or training mono network by both KITTI and Cityscapes datasets?

**Comment by xy-guo** [comment]
You can use the StereoNoFt model to pretrain the monocular model on Cityscapes. Remember to cut off the bottom part for the Cityscapes dataset.

**Comment by sazippe** [comment]
I want to know the flow of the following training.
StereoUnsupFt→Mono pt No S,K→C,K

Is this flow correct?
First, train stereo model on scenflow and then finetune with KITTI.
Next, train mono model (pt with VGG) on Cityspace and then finetune with KITTI.

**Comment by xy-guo** [comment]
Yes

**Comment by sazippe** [comment]
Thank you!
Training list is 'list/cityscapes_train_list.txt', right?

**Comment by xy-guo** [comment]
Yes. Note that you should supervise the mono model (pt with VGG) on Cityscapes with StereoNoFt and then finetune on KITTI with StereoUnsup/SupFt model. The learning rate is not changed for finetuning.

**Comment by sazippe** [comment]
Thank you for your advice.
